Character.cpp

```diff
@@ -4,37 +4,88 @@
 #include "Character.h"
 #include "JsonParser.h"
 
-Character::Character(std::string name, int HP, int DMG) : characterName(name), characterHP(HP), characterDMG(DMG) {}
+Character::Character(std::string name /** This is a string parameter*/, int HP /** This is an int parameter*/, int DMG /** This is an int parameter*/, double ACD /** This is a double parameter*/) : characterName(name), characterHP(HP), characterDMG(DMG), characterACD(ACD) {}
```

Comment:
Ilyen súgónak én nem látom sok gyakorlati hasznát.

Reply:
@wajzy Mi se láttuk túl sok értelmét, de mivel a videóban szerepelt, ezért beleírtuk őket mi is. Ha nincs rájuk szükség, nagyon szívesen kivesszük őket a következő feladatnál, ha @hegyhati es @vizvezetek es egyet ertenek ezzel.

Reply:
Hat a tipusat leirni valoban semmi ertelme. Az ertelmet mar tobb. Nyilvan egy jo kodban alig van szukseg barmilyen doxy kommentre, itt most a gyakorlas erdekeben kicsit atesunk a lo tuloldalara. De ez mar Ausztraliaba estunk ;-P